core/src/main/scala/org/apache/spark/scheduler/TaskSchedulerImpl.scala

```diff
@@ -332,7 +332,8 @@ private[spark] class TaskSchedulerImpl(
           // We lost this entire executor, so remember that it's gone
           val execId = taskIdToExecutorId(tid)
           if (activeExecutorIds.contains(execId)) {
-            removeExecutor(execId)
+            removeExecutor(execId, SlaveLost(s"Task $tid was lost, so" +
```

Comment:
Minor, but I find it slightly more readable if you avoid breaking the string:

            removeExecutor(execId,
              SlaveLost(s"Task $tid was lost, so marking the executor as lost as well."))